server/models/Order.js

```diff
@@ -446,6 +447,10 @@ function defineModel() {
     });
   };
 
+  Order.cancelActiveOrdersByTierId = function (tierId) {
+    return Order.update({ status: OrderStatuses.CANCELLED }, { where: { TierId: tierId } });
+  };
+
```

Comment:
See the feedback in https://github.com/opencollective/opencollective-api/pull/8004#discussion_r983179952